### ISSUE TITLE
Subid

### ISSUE THREAD

**Issue by JacondaMarsi** [dengivseti/WhalesTracker#13]
Возможно ли реализовать подсчет уников, не за счет IP , а за счет например своего параметра например subid, который бы переодевался по ссылке в трекере, что то типа: domen.com/xcxnjss?subid=dfhhee321

**Comment by dengivseti** [comment]
Да. Можно указать query параметр и он будет отображаться в статистике, подефолту этот параметр равен q. Но его можно поменять на любой другой в разделе global settings. Подробнее можете прочитать здесь https://whalestracker.netlify.app/started/statistick
